.pylintrc

```diff
@@ -22,8 +21,15 @@ disable=
   too-many-nested-blocks,
   too-many-public-methods,
   too-many-statements,
-  too-public-methods,
   wrong-import-order,
+  import-error,
+  consider-using-f-string,
+  use-implicit-booleaness-not-comparison,
+  unspecified-encoding,
+  no-name-in-module,
+  use-list-literal,
+  use-dict-literal,
```

Comment:
`import-error`, `no-name-in-module`: pylint is no longer installed with all project dependencies, checks like this is better left to mypy + runtime tests we already have in place.
`consider-using-f-string`, `use-list-literal`, `use-dict-literal`: Covered by [pyupgrade](https://github.com/aiven/karapace/pull/530).
`unspecified-encoding`: Covered by pyupgrade, but with different resolution.
`use-implicit-booleaness-not-comparison`: I think this is overly opinionated. `foo == []` is fine and doesn't need to be outlawed.